examples/metal-2-fabric-aws-connection/versions.tf

```diff
@@ -5,5 +5,9 @@ terraform {
       source  = "equinix/equinix"
       version = ">= 1.20.0"
```

Comment:
All the equinix versions need to be updated to leverage the bug fix that was made to ensure the AWS provider could be integrated into the examples. I believe it's 1.25.1.